crates/host/src/wasmbus/mod.rs

```diff
@@ -1426,23 +1355,11 @@ impl ActorInstance {
         };
 
         // actors don't have a contract_id
-        let target_public_key = invocation.target.public_key;
-        let target = if invocation.target.contract_id.is_empty() {
-            let actor_claims = self.actor_claims.read().await;
-            let claims = actor_claims
-                .get(&target_public_key)
-                .cloned()
-                .context("failed to look up claims for target")?;
-            PolicyRequestTarget::from(claims)
-        } else {
-            let provider_claims = self.provider_claims.read().await;
-            let claims = provider_claims
-                .get(&target_public_key)
-                .cloned()
-                .context("failed to look up claims for target")?;
-            let mut target = PolicyRequestTarget::from(claims);
-            target.link_name = Some(invocation.target.link_name.clone());
-            target
+        let target = PolicyRequestTarget {
+            public_key: None, // todo(vadossi-cosmonic): get this from somewhere (pubkey of actor/provider)
+            issuer: None, // todo(vadossi-cosmonic): get this from somewhere (target claim issuer)
+            contract_id: None, // todo(vadossi-cosmonic): get this from somewhere (contract ID)
+            link_name: Some(invocation.target.link_name.clone()),
```

Comment:
FWIW, we're gonna have to reshape the policy request format to deal with not having claims information everywhere necessarily

Reply:
Ah, so can I remove those TODOs? Are we going to get none of that information from there?